site/_posts/2023-08-03-announcing-fusionauth-1-47.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: blog-post
+title: Announcing FusionAuth 1.47
+description: This release includes performance improvements, the ability to include preferred languages on the basic registration form, and SAMLv2 assertion encryption.
+author: Dan Moore
+image: blogs/release-1-47/fusionauth-1-47.png
+category: announcement
+tags: release-announcement localization registration performance lambda connector http metrics saml
+excerpt_separator: "<!--more-->"
+---
+
+FusionAuth version 1.47 shipped in late July, 2023. This release includes performance improvements, the ability to include preferred languages on the basic registration form, and SAMLv2 assertion encryption.
+
+<!--more-->
+
+The focus of this release was improving performance. In fact, I hereby dub it the "Performance panther" release.
+
+All in all there are 21 issues, enhancements, and bug fixes included in the 1.47 releases. As always, please see the [release notes](/docs/v1/tech/release-notes#version-1-47-1) for a full breakdown of the changes between 1.46 and 1.47, including any schema changes.
```

Comment:
```suggestion
All in all there are 21 issues, enhancements, and bug fixes included in the 1.47 release. As always, please see the [release notes](/docs/v1/tech/release-notes#version-1-47-1) for a full breakdown of the changes between 1.46 and 1.47, including any schema changes.
```

Reply:
I am using the term "releases" because it includes 1.47.0 and 1.47.1. Not sure of a clean way to do this. Welcome to suggestions.

Reply:
That is what I originally thought, but got confused when I read the singular "The focus of this release was improving performance. In fact, I hereby dub it the "Performance Panther" release."  Releases is correct so you may want to change line 18 back to releases and then say something like "The focus of these **updates** was improving performance."  Then the "I herby dub"... as a singular release makes sense.  
I also think "The focus of **these releases** was improving performance.  In fact, I hereby dub them the "Performance Panther" release." would work.